README.md

```diff
@@ -10,99 +10,102 @@
 
 ## Comparison with Mirror
-When migrating a project from Mirror to Mirage, these will be the most notable differences.
+
+Mirage has some notable differences from its distant sister, Mirror. The table below briefly details them:
 
 | Mirage                                              | Mirror                                 |
 | --------------------------------------------------- | -------------------------------------- |
-| Install via Unity Package Manager                   | Install from Asset Store               |
+| Installs via Unity Package Manager                  | Installs from Asset Store              |
 | Errors are thrown as exceptions                     | Errors are logged                      |
 | `[ServerRpc]`                                       | `[Command]`                            |
-| `[ClientRpc(target=Client.Owner)]`                  | `[TargetRpc]`                          |
+| `[ClientRpc(target = Client.Owner)]`                | `[TargetRpc]`                          |
 | Subscribe to events in `NetworkServer`              | Override methods in `NetworkManager`   |
 | Subscribe to events in `NetworkClient`              | Override methods in `NetworkManager`   |
 | Subscribe to events in `NetworkIdentity`            | Override methods in `NetworkBehaviour` |
 | Methods use PascalCase (C# guidelines)              | No consistency                         |
 | `NetworkTime` available in `NetworkBehaviour`       | `NetworkTime` is global static         |
 | Send any data as messages                           | Messages must implement NetworkMessage |
-| Supports Unity 2019.3 or later                      | Supports Unity 2018.4 or later         |
+| Supports Unity 2020 LTS or later                    | Supports Unity 2019 LTS or later       |
 | Offers simple Socket API to implement new protocols | Each protocol requires a new transport |
 
-Mirage has many new features
-* Mirage supports [fast domain reload](https://blogs.unity3d.com/2019/11/05/enter-play-mode-faster-in-unity-2019-3/)
+**Some notable features that Mirage has:**
+
+* [Fast play mode support](https://blogs.unity3d.com/2019/11/05/enter-play-mode-faster-in-unity-2019-3/)
+* Clients can connect to multiple servers - for example, a chat server while connected to a game server.
 * Components can be added in child objects
-* Your client can connect to multiple servers. For example chat server and game server
-* Modular,  use only the components you need.
+* Modular API: You only use the components you need.
 * Error handling
 * [Version defines](https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html#define-symbols)
 * Server Rpcs can [return values](https://miragenet.github.io/Mirage/Articles/Guides/RemoteCalls/ServerRpc.html)
-* The default transport features DoS prevention
-* The default transport has CRC64 integrity check
+* A built-in reliable UDP default transport featuring DoS prevention and CRC64 integrity checks
```

Comment:
I'm not sure the "featuring DoS prevention and CRC64 integrity checks" is in the new transport, we should probably remove this part for now.

And make we can open a separate github issue if people want the CRC64 integrity stuff

Reply:
Alright, will remove it - I believe that was a fragment from the old readme.

Reply:
Removed in the latest commit.